### ISSUE TITLE
Move the configuation & init scripts to the Dockerfile

### ISSUE THREAD

**Issue by dktsni** [Decathlon/ara#115]
# Move the configuation & init scripts to the Dockerfile

## Context

For local testing and PoC, a docker container of Mysql can be created, using the `manage-db.sh` script. 3 volumes are mounted to the container, one for the datas persistency, and 2 for configuration and init scripts. This leads to several problems: 

* The configuration and init scripts mount are unecessary, since they're executed only once at the first boot of the container. 
* On several Windows systems, Docker randomly refuse to mount the volumes, which prevent the custom configuration and databases for ARA to be set, and prevents the app from starting
* The launch script is a Shell script, and it force the user on Windows to either install a WSL or a Git Bash.

## Expected

* Move the init script and config script to the Dockerfile itself to set up the container once for all the systems.
* Remove the need of a mounted volume for datas persistency, since the docker container is not recommanded for production, we don't need to persists any datas
* Adapt the Shell script for Windows used without installing any other tools than Docker (possible duplicate of #93 )
